tests/bluetooth/controller/ctrl_encrypt/CMakeLists.txt

```diff
@@ -0,0 +1,74 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.13.1)
+
+#
+# the following allows us to test the different PHY configurations
+# by setting the appropriate flags in the yaml file
+#
+if(CONFIG_BT_CTLR_PHY)
+	add_compile_definitions(CONFIG_BT_CTLR_PHY)
+endif(CONFIG_BT_CTLR_PHY)
+
+if(CONFIG_BT_CTLR_PHY_2M)
+	add_compile_definitions(CONFIG_BT_CTLR_PHY_2M)
+endif(CONFIG_BT_CTLR_PHY_2M)
+
+if(CONFIG_BT_CTLR_PHY_CODED)
+	add_compile_definitions(CONFIG_BT_CTLR_PHY_CODED)
+endif(CONFIG_BT_CTLR_PHY_CODED)
+
+if(CONFIG_BT_CTLR_DATA_LENGTH_MAX)
+	add_compile_definitions(CONFIG_BT_CTLR_DATA_LENGTH_MAX=${CONFIG_BT_CTLR_DATA_LENGTH_MAX})
+endif(CONFIG_BT_CTLR_DATA_LENGTH_MAX)
```

Comment:
Could we remove this?
Similar in the other CMakeLists.txt files.

Reply:
Yes we can, but I want to keep at least an example easily available, for when we want to use this trick.

Reply:
I'll remove it here, and add something to the feature exchange unit tests

Reply:
Fine with me